### ISSUE TITLE
Improve Voiceover accessibility

### ISSUE THREAD

**Issue by chrisnestrud** [horizontalsystems/unstoppable-wallet-ios#4228]
Please consider improving accessibility of this app when using Voiceover, the screen-reader included with iOS.

Examples of issues:

On the Balance tab, after selecting an item such as BTC, the Send and Receive buttons are not visible to Voiceover. They could be added to the Voiceover Actions rotor; there is currently an option labeled circle_minus_shifted_24 there.

On the Balance tab, Voiceover focus gets stuck. Focus starts at the beginning of the controls and hits 'Switch Wallet 24' button, 'NFT24' button, Wallet name heading, '5 stars' button, then keeps visiting '5 stars' button, 'Balance' button, etc. and does not move to the list of coins.

Some other cosmetic naming issues for controls.

References:
https://developer.apple.com/documentation/accessibility/supporting_voiceover_in_your_app

Thanks for your consideration, please let me know if I can provide any further information.
